pyvista/core/utilities/features.py

```diff
@@ -213,7 +213,7 @@ def voxelize(
         del selection
 
     # extract cells from point indices
-    return ugrid.extract_points(mask)
+    return ugrid.extract_points(mask)  # type: ignore[arg-type]
```

Comment:
return type is `NumpyArray[np.bool_]`, but `extract_points` expects `NumypyArray[bool]`. Might be time to update `npt-promote` to fix this (it current only promotes float/np.float_, int/np.int_, not bool)